backend-models/service-tests/profileservice-tests.txt.txt

```diff
@@ -0,0 +1,232 @@
+profileservice-tests.txt
+
+
+Before tests start, create 2 dummy accounts, one as teacher, another as student. Examples for names, 
+name = TestSTUD, PASSW = testpasswS
+name = TestTEACH, PASSW = testpasswT
+
+
+
+--------------------------------------------
+Row : 46 Check if it blocks teacher
+--------------------------------------------
+---------------------
+Name : teacherBlockerTEST
+---------------------
+
+*********
+* pre-req:
+********* 
+pdoConnect();
+session_start();
+
+if(isset($_SESSION['uid'])){
+	$userid=$_SESSION['uid'];
+}else{
+	$userid="1";		
+} 
+
+*********
+End pre-
+********* 
+
+if(checklogin()) {
```

Comment:
These are supposed to be independent of language, which this isn't, it has alot of copy-pasted code from the file which it also isn't really supposed to have. What we need really, is to provide values for the file to be able to perform intended function. And the pre-requisites needed to get there.

I'm comparing with the accepted test-file we have so far: https://github.com/HGustavs/LenaSYS/blob/master/backend-models/service-tests/duggaedservice-tests.txt